custom_components/tahoma/climate_devices/somfy_heating_temperature_interface.py

```diff
@@ -0,0 +1,220 @@
+"""Support for Atlantic Electrical Heater."""
+import logging
+from typing import Optional
+
+from homeassistant.components.climate import (
+    SUPPORT_PRESET_MODE,
+    SUPPORT_TARGET_TEMPERATURE,
+    ClimateEntity,
+)
+from homeassistant.components.climate.const import (
+    CURRENT_HVAC_COOL,
+    CURRENT_HVAC_HEAT,
+    HVAC_MODE_AUTO,
+    HVAC_MODE_HEAT_COOL,
+    HVAC_MODE_OFF,
+    PRESET_AWAY,
+    PRESET_COMFORT,
+    PRESET_ECO,
+    PRESET_NONE,
+)
+from homeassistant.const import (
+    ATTR_TEMPERATURE,
+    EVENT_HOMEASSISTANT_START,
+    STATE_UNKNOWN,
+    TEMP_CELSIUS,
+)
+from homeassistant.core import callback
+from homeassistant.helpers.event import async_track_state_change
+
+from ..coordinator import OverkizDataUpdateCoordinator
+from ..entity import OverkizEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+COMMAND_SET_HEATING_LEVEL = "setHeatingLevel"
+
+CORE_OPERATING_MODE_STATE = "ovp:HeatingTemperatureInterfaceActiveModeState"
+CORE_TARGET_TEMPERATURE_STATE = "core:TargetTemperatureState"
+CORE_ON_OFF_STATE = "core:OnOffState"
+IO_TARGET_HEATING_LEVEL_STATE = "io:TargetHeatingLevelState"
+
+
+TAHOMA_TO_PRESET_MODES = {
+    "secured": PRESET_AWAY,
+    "eco": PRESET_ECO,
+    "comfort": PRESET_COMFORT,
+    "free": PRESET_NONE,
+}
+
+PRESET_MODES_TO_TAHOMA = {v: k for k, v in TAHOMA_TO_PRESET_MODES.items()}
+
+TAHOMA_TO_HVAC_MODES = {
+    "auto": HVAC_MODE_AUTO,
+    "manu": HVAC_MODE_HEAT_COOL,
+}
+
+HVAC_MODES_TO_TAHOMA = {v: k for k, v in TAHOMA_TO_HVAC_MODES.items()}
+
+TAHOMA_TO_HVAC_ACTION = {"cooling": CURRENT_HVAC_COOL, "heating": CURRENT_HVAC_HEAT}
```

Comment:
We need to add them to `OverkizCommandParam` indeed, as you mentioned. For now this is fine :).